src/splitInteger.test.js

```diff
@@ -1,21 +1,27 @@
 'use strict';
 
-const splitInteger = require('./splitInteger');
-
-test(`should split a number into equal parts
-  if a value is divisible by a numberOfParts`, () => {
-
-});
-
-test(`should return a part equals to a value
-  when splitting into 1 part`, () => {
-
-});
-
-test('should sort parts ascending if they are not equal', () => {
-
-});
-
-test('should add zeros if value < numberOfParts', () => {
-
+describe('splitInteger', () => {
+  const splitInteger = require('./splitInteger');
```

Comment:
Move the `require` statement to the top of the file, outside the `describe` block, for better readability and consistency with common practices.